test/e2e/cypress/integration/single-backend/environments.spec.js

```diff
@@ -499,6 +437,107 @@ describe('Environments', function() {
     cy.title().should('eq', 'localEnvTestTabTitle')
   })
 
+  it.skip('Should be able to switch to a reachable environment without lazy loading sequence error', function() {
```

Comment:
Why are we skipping this one ?

Reply:
Because it is the one that breaks the Kuzzle V1 security system. @Njuelle will take care of rewriting this test later on.